JPET_GATE_Use_Examples/README.md

```diff
@@ -1,107 +1,113 @@
-About this section
-------------------
-
-Here you can find some examples how work with GATE by using macros. Each one example focus on other problem.
-
-How to correctly install GATE from this repository?
----------------------------------------------------
+# Required tools
+Before you start do anything be sure you have installed below listed tools:
+ - cmake >= 3.10.2
+ - ccmake >=3.10.2
+ - gcc >= 7.3.0
```

Comment:
Isn't this requirement a little too tight? ROOT, Geant and Gate all compile with gcc v5

Reply:
No exactly - indeed you can compile with gcc v5 but this version is really old (from 2014-15). In this period of time gcc increased its compatibility with c++14. Moreover, I'm sure we will start using some solutions from c++17 to make code works better. In this case we will omit potential problems with compilation in  the nearest future.